src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
```

Comment:
It seems like you're using flexbox to layout the elements within the card. However, since you're using 'flex-direction: column', the default display of 'block' would have the same effect. Consider removing the 'display: flex' and 'flex-direction: column' properties unless you plan to use other flexbox features.